e2e/ansible/playbooks/hyperconverged/test-resize-jiva-volume/resize-volume.yml

```diff
@@ -0,0 +1,281 @@
+          shell: kubectl get nodes -o wide  | awk {'print $1'} | awk 'FNR == 4{print}'
+          args:
+            executable: /bin/bash
+          register: node_name
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+  
+        - name: Creating label for the node
+          shell: kubectl label nodes {{node_name.stdout}} app=percona
+          args:
+            executable: /bin/bash
+          register: label
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+          until: '"labeled" in label.stdout'
+          delay: 10
+          retries: 3
+
+        - name: Deploying application
+          shell: source ~/.profile; kubectl apply -f "{{ percona_def }}"
+          args:
+            executable: /bin/bash
+          register: percona_status
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+          until: "'deployment \"percona\" created' in percona_status.stdout"
+          delay: 30
+          retries: 5
+        
+        - name: Checking if the application is deployed successfully
+          shell: kubectl get pods --all-namespaces | grep percona
+          args:
+            executable: /bin/bash
+          register: app_status
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+          until: "'Running' in app_status.stdout"
+          delay: 30
+          retries: 15
+    
+        - name: Getting PV name
+          shell: kubectl get pv | grep openebs-percona | awk {'print $1'}
+          args:
+            executable: /bin/bash
+          register: pv
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         
+        - name: Getting the controller service IP address
+          shell: kubectl get svc | grep "{{ pv.stdout }}" | grep ctrl | awk {'print $3'}
+          args:
+            executable: /bin/bash
+          register: target_IP
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+         
+        - name: Getting the device name
+          shell: fdisk -l | grep "5 G" | awk {'print $2'} | sed 's/.$//'
+          args: 
+            executable: /bin/bash
+          become: true
+          register: device_name
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+ 
+        - name: Getting the mounted directory
+          shell: mount | grep "{{ device_name.stdout }}" | awk {'print $3'} | awk 'FNR == 1 {print}'
+          args:
+            executable: /bin/bash
+          register: mount_path
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+
+        - name: Unmount the file system
+          mount:
+            name: "{{ mount_path.stdout }}"
+            state: unmounted
+          become: true
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+         # shell: umount -l "{{ mount_path.stdout }}"
+         # args:
+         #   executable: /bin/bash
+         # become: true
+         # register: unmount
+         # delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
```

Comment:
Let us remove the commented steps as a best-practice unless it has a significant purpose

Reply:
Removed